alts/src/main/java/io/grpc/alts/internal/AltsHandshakerStub.java

```diff
@@ -69,7 +69,7 @@ public HandshakerResp send(HandshakerReq req) throws InterruptedException, IOExc
     if (!result.isPresent()) {
       maybeThrowIoException();
     }
-    return result.get();
+    return result.orNull();
```

Comment:
This is wrong. Right now if the result is missing it will throw; this is not a nullable response. With this change (along with a bug) it could start returning null which the caller (AltsHandshakerClient) isn't prepared for.

The code isn't all that good about how it "guarantees" the result is present, so I'm not saying there isn't an improvement to make here, but swapping to `onNull()` seems wrong.

Reply:
Fixed to throw an IOException with meaningful message when result isn't present and no other exception was seen.